onset_maproom/docker/app.wsgi

```diff
@@ -0,0 +1,2 @@
+import maproom
+application = maproom.APP.server
```

Comment:
Unrelated to your PR, allow me to vent my frustration at dash for basing their product on flask but refusing to use flask's terminology. The thing flask calls an "app" gets renamed a "server" by dash, and dash invents its own thing which it calls "app".